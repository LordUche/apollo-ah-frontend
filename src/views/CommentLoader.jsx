import React from 'react';

export default function CommentLoader() {
  return (
    <div className="comment-loader">
      <div className="pg-empty-placeholder comment-loader-img" />
      <div className="pg-empty-placeholder comment-loader-name" />
      <div className="pg-empty-placeholder comment-loader-name comment-loader-time" />
      <div className="pg-empty-placeholder comment-loader-name comment-loader-text" />
      <div className="pg-empty-placeholder comment-loader-name comment-loader-text comment-loader-text-sm" />
    </div>
  );
}
