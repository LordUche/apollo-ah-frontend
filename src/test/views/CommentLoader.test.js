import React from 'react';
import { shallow } from 'enzyme';
import CommentLoader from '../../views/CommentLoader';


describe('<CommentLoader>', () => {
  it('should render without crashing', () => {
    const wrapper = shallow(<CommentLoader />);
    expect(wrapper).toMatchSnapshot();
  });
});
