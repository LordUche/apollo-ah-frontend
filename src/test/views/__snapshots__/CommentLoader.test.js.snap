// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CommentLoader> should render without crashing 1`] = `
<div
  className="comment-loader"
>
  <div
    className="pg-empty-placeholder comment-loader-img"
  />
  <div
    className="pg-empty-placeholder comment-loader-name"
  />
  <div
    className="pg-empty-placeholder comment-loader-name comment-loader-time"
  />
  <div
    className="pg-empty-placeholder comment-loader-name comment-loader-text"
  />
  <div
    className="pg-empty-placeholder comment-loader-name comment-loader-text comment-loader-text-sm"
  />
</div>
`;
